pyproject.toml

```diff
@@ -14,7 +14,6 @@ keywords = [
   "isolated",
   "virtual",
 ]
-license = "MIT"
```

Comment:
This is correct according to the newer standards. The version Field is optional because can be dynamic as is here. It sounds to me like poetry has some broken checks.

Reply:
@gaborbernat Are you saying that https://packaging.python.org/en/latest/specifications/pyproject-toml/#license is not up to date?

Reply:
Yes, and no. This is a specification that is in draft, so not yet accepted however our build back end already uses it.